src/AppBundle/Command/ConvertMJMLThemeCommand.php

```diff
@@ -91,14 +91,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
         /** @var SplFileInfo $mjmlFile */
         foreach ($finder as $mjmlFile) {
             //Ignore components file for now
+            $path_separator = preg_match('#/#', $mjmlFile->getRelativePathname()) ? '/' : '\\';
             if (preg_match('/^components/', $mjmlFile->getRelativePathname())) {
-                if ('components/layout.mjml.twig' == $mjmlFile->getRelativePathname() ||
-                    'components/order_layout.mjml.twig' == $mjmlFile->getRelativePathname()) {
+                if ('components' . $path_separator . 'layout.mjml.twig' == $mjmlFile->getRelativePathname() ||
```

Comment:
Is there an equivalent of DIRECTORY_SEPARATOR that would be usable here?

Reply:
I thought, safest would be to rely on the content of `$mjmlFile->getRelativePathname()` rather than `DIRECTORY_SEPARATOR`, even though both are identical in most situations.